pyproject.toml

```diff
@@ -39,7 +39,7 @@ keywords = [
     "ascii",
 ]
 dependencies = [
-    "numpy>=1.23",
+    "numpy>=1.23", # SPEC 0
```

Comment:
What is even nicer is that if Scientific Python has a script for downstream to spit out what are the expected versions of stuff that complies to SPEC 0. 👀